docs/Coding-Guidelines/SelfContained.md

```diff
@@ -0,0 +1,46 @@
+# Self-Contained
+
+## What is 'Self-Contained'?
+
+Windows App SDK can be deployed via MSIX or self-contained, aka WinAppSDK/MSIX and WinAppSDK/SelfContained.
```

Comment:
Nit: Doesn't answer the question -- what is self-contained?

Reply:
"Self-Contained" is the phrase for what was formerly referred to as "AppLocal" or "In-App".

In short, Windows App SDK deployed as Self-Contained means "here's the files, add them to your add". No Microsoft.WindowsAppSDK MSIX packages.

The Self-Contained spec is in-progress at [Initial draft of AppLocal Deployment spec. #1202](https://github.com/microsoft/WindowsAppSDK/pull/1202)

+@mikebattista